aggregator.py

```diff
@@ -213,12 +215,14 @@ def flush(self, timestamp, interval):
 class Histogram(Metric):
     """ A metric to track the distribution of a set of values. """
 
-    def __init__(self, formatter, name, tags, hostname, device_name):
+    def __init__(self, formatter, name, tags, hostname, device_name,
+            aggregates=None, percentiles=None):
         self.formatter = formatter
         self.name = name
         self.count = 0
         self.samples = []
-        self.percentiles = [0.95]
+        self.aggregates = aggregates or []
+        self.percentiles = percentiles or [0.95]
```

Comment:
I can get rid of this default, actually I made a mistake in this code above (aggregates should be initialized to the old default). I hope it's not the case but if people rely on this `Histogram` class we should be careful about changing the default init...